gemfiles/activerecord_5.2.gemfile.lock

```diff
@@ -61,7 +60,6 @@ DEPENDENCIES
   rake
   rspec (~> 3.9)
   sqlite3
-  wwtd
 
 BUNDLED WITH
    1.17.3
```

Comment:
afaik this will crash and is why I added the matching_bundler

Reply:
In my experience, ruby/setup-ruby handles installing the correct bundler version.

Reply:
if it works then that's great :D